plugins/dubbo/src/test/java/com/megaease/easeagent/plugin/dubbo/interceptor/trace/alibaba/AlibabaDubboTracingInterceptorTest.java

```diff
@@ -0,0 +1,133 @@
+import com.megaease.easeagent.plugin.enums.Order;
+import com.megaease.easeagent.plugin.field.AgentFieldReflectAccessor;
+import com.megaease.easeagent.plugin.interceptor.MethodInfo;
+import org.apache.dubbo.rpc.AppResponse;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import java.util.concurrent.CompletableFuture;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+@RunWith(EaseAgentJunit4ClassRunner.class)
+public class AlibabaDubboTracingInterceptorTest extends AlibabaDubboBaseTest {
+
+	private AlibabaDubboTracingInterceptor alibabaDubboTracingInterceptor;
+
+	@Before
+	public void setup() {
+		super.setup();
+		this.alibabaDubboTracingInterceptor = new AlibabaDubboTracingInterceptor();
+	}
+
+	@Test
+	public void init() {
+		DubboPlugin dubboPlugin = new DubboPlugin();
+		InterceptorTestUtils.init(alibabaDubboTracingInterceptor, dubboPlugin);
+		assertNotNull(AgentFieldReflectAccessor.getStaticFieldValue(AlibabaDubboTracingInterceptor.class, "DUBBO_TRACE_CONFIG"));
+	}
+
+	@Test
+	public void rpcConsumerCallSuccess() {
+		MethodInfo methodInfo = MethodInfo.builder()
+				.args(new Object[]{consumerInvoker, consumerInvocation})
+				.retValue(this.successResult)
+				.build();
+
+		Context context = EaseAgent.getContext();
+		DubboPlugin dubboPlugin = new DubboPlugin();
+		InterceptorTestUtils.init(alibabaDubboTracingInterceptor, dubboPlugin);
+		alibabaDubboTracingInterceptor.before(methodInfo, context);
+		alibabaDubboTracingInterceptor.after(methodInfo, context);
+		this.assertConsumerTrace(successResult.getValue(), null);
+	}
+
+	@Test
+	public void rpcProviderCallSuccess() {
+		MethodInfo methodInfo = MethodInfo.builder()
+				.args(new Object[]{providerInvoker, providerInvocation})
+				.retValue(this.successResult)
+				.build();
+
+		Context context = EaseAgent.getContext();
+		DubboPlugin dubboPlugin = new DubboPlugin();
+		InterceptorTestUtils.init(alibabaDubboTracingInterceptor, dubboPlugin);
+		alibabaDubboTracingInterceptor.before(methodInfo, context);
+		alibabaDubboTracingInterceptor.after(methodInfo, context);
+		this.assertProviderTrace(successResult.getValue(), null);
+	}
+
+	@Test
+	public void rpcConsumerAsyncCallSuccess() {
+		MethodInfo methodInfo = MethodInfo.builder()
+				.args(new Object[]{asyncConsumerInvocation.getInvoker(), asyncConsumerInvocation})
+				.build();
+
+		AppResponse appResponse = new AppResponse(successResult);
+		CompletableFuture<AppResponse> completeFuture = new CompletableFuture<>();
+		RpcContext.getContext().setFuture(completeFuture);
+		Context context = EaseAgent.getContext();
+		DubboPlugin dubboPlugin = new DubboPlugin();
+		InterceptorTestUtils.init(alibabaDubboTracingInterceptor, dubboPlugin);
+		alibabaDubboTracingInterceptor.before(methodInfo, context);
+		alibabaDubboTracingInterceptor.after(methodInfo, context);
+		completeFuture.complete(appResponse);
```

Comment:
When you test an asynchronous call, you should let it execute asynchronously, such as creating a new thread to run it